src/test/common/steps/confirm-details.js

```diff
@@ -31,3 +32,14 @@ async function checkAllPageContentIsPresentAndCorrect () {
   const panelTitle = await pages.confirm.getPanelTitleText()
   expect(panelTitle.toString().trim()).to.be.equal('Application complete', 'expected confirmation header to be correct')
 }
+
+async function assertVoucherEntitlementValuesForAttr (attr, expectedValue) {
+  try {
+    const message = `expected all references to ${attr.replace('-', ' ')} to be correct`
+    const refs = await pages.confirm.getVoucherEntitlementRefsForDataAttr(attr)
+    const incorrectAmounts = refs.filter(value => value !== expectedValue)
+    expect(incorrectAmounts.length).to.be.equal(0, message)
```

Comment:
Is there a `expected(incorrectAmounts).to.be.empty(message)` that we can use here instead?